src/lib/env.ts

```diff
@@ -1,5 +1,7 @@
 import express from 'express';
 
+// WARNING - do not use these in the modules, they are server-side only
```

Comment:
future refactor opportunity?
```
/modules
/server
/shared
```

Reply:
yes!!!

Reply:
I've added a card

Reply:
I had a little look into this on a learning day a few weeks back - I think I'll revisit it tomorrow! Still just at the stage of spiking out how it would look.